### PR TITLE
index.html: add inf oabout v4.0.16 and v4.6.8

### PR DESCRIPTION
**Pull request by miczyg1**
Signed-off-by: Michał Żygowski <michal.zygowski@3mdeb.com>

### CODE REVIEW

**Comment by miczyg1** [comment]
@mscibisz please check if formatting is ok

**Comment by mscibisz** [comment]
I have fixed it up a bit. The first versions (mainline and legacy) contain div "container" which is meant only for the very first, so it needed to be moved to the new version only. Also, I've added a border-bottom to the ex-first versions to separate it from the newly added versions.

File: index.html [hunk 1]
```diff
@@ -91,22 +91,27 @@ <h2>OS status</h2>
...
                     <td>pfSense</td>
                     <td>OK</td>
-                    <td><sup>OK(1)</sup></td>
+                    <td><sup>(1)</sup></td>
                     <td>not tested</td>
                     <td><sup>(2)</sup></td>
                 </tr>
             </table>
 
-            <p><sup>(1)</sup> - works well with BIOS v4.6.8 or later in mainline and all legacy releases</p>
```
Comment (miczyg1) [inline index.html]: @mscibisz please roll it back. It is a part of update.

File: index.html [hunk 2]
```diff
@@ -149,16 +154,15 @@ <h4><a target="_blank" href="https://github.com/pcengines/coreboot/compare/v4.6.
...
-
+            </div>
 
 <!-- mainline releases v4.6.7 -->
-        <div class="break"></div>
-        <div class="container" id="mr-7">
-            <div class="smaller-margin">
+            <div class="break"></div>
+            <div class="smaller-margin" id="mr-7">
+                <h2 class="text-middle">Mainline releases</h2>
```
Comment (miczyg1) [inline index.html]: @mscibisz why did You put title again here?

File: index.html [hunk 3]
```diff
@@ -487,56 +491,60 @@ <h4><a target="_blank" href="https://github.com/pcengines/coreboot/compare/v4.6.
...
             </div>
-        </div>
 
 <!-- legacy releases v4.0.15 -->
-        <div class="break"></div>
-        <div class="container" id="lr-7">
-            <div class="smaller-margin">
+            <div class="break"></div>
+            <div class="smaller-margin" id="lr-7">
+                <h2 class="text-middle">Legacy releases</h2>
```
Comment (miczyg1) [inline index.html]: @mscibisz again repeated title. Why?

**Comment by miczyg1** [comment]
@mscibisz thanks for help. You have overlooked few things. Please fix it

**Comment by pietrushnic** [comment]
@mscibisz @miczyg1 please let me know if there is anything to fix here.

**Comment by miczyg1** [comment]
@pietrushnic looks allright to me. @mscibisz anything to add?

**Comment by mscibisz** [comment]
It's all fine

**Pull request merged**